webapp/src/images/icons/sign.jsx

```diff
@@ -2,20 +2,8 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import {SignInIcon} from '@primer/octicons-react';
 
-export default class SignIcon extends React.PureComponent {
-    render() {
-        return (
-            <svg
-                role='img'
-                viewBox='0 0 512 512'
-                width='16'
-                height='16'
-            >
-                <path
-                    d='M496 64H128V16c0-8.8-7.2-16-16-16H80c-8.8 0-16 7.2-16 16v48H16C7.2 64 0 71.2 0 80v32c0 8.8 7.2 16 16 16h48v368c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V128h368c8.8 0 16-7.2 16-16V80c0-8.8-7.2-16-16-16zM160 384h320V160H160v224z'
-                />
-            </svg>
-        );
-    }
-}
+export default () => (
+    <SignInIcon size='small'/>
```

Comment:
I think you want `MilestoneIcon` instead 
![Screenshot from 2020-10-01 14-35-43](https://user-images.githubusercontent.com/16541325/94809799-6ae10480-03f3-11eb-9b11-c73247098ecb.png)